src/style.css

```diff
@@ -1,3 +1,53 @@
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.logo {
+  margin-left: 50px;
+  padding-top: 10px;
+}
+
+.main_nav {
+  display: flex;
+  gap: 20px;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 60px;
+  text-transform: uppercase;
+  margin-right: 50px;
+
+  height: 60px;
+}
```

Comment:
Avoid using tag names for styling. It's better to use classes to style elements to prevent issues with specificity and reusability. Consider refactoring the `.main_nav` class to avoid styling the `a` tag directly, and instead use a class like `.main_nav-link`.